Update 03_riferimenti-sigle.rst (#225)
italia/lg-modellointeroperabilita-docs: doc/00_Linee guida interoperabilità tecnica/03_riferimenti-sigle.rst

     -    Application Programming Interface

   * -    [API-First]
     - L'API-first è un approccio in cui le PA considerano le API 
       come mezzo principale per perseguire i propri obiettivi, 
       interagendo con i propri stakeholder sin dalla fase di 
       progettazione.   
       Come indica il CAD art. 64-bis, comma 1-bis quindi,
       le interfacce applicative (API appunto) devono essere progettate
       e/o evolute in maniera interoperabile,
       a prescindere dai canali di erogazione del servizio 
       che sono individuati logicamente e cronologicamente 

>        dopo la progettazione dell’API.

Reviewer: ```suggestion
       dopo la progettazione dell’API
```

     -    Application Programming Interface

   * -    [API-First]

>      -    Strategia di sviluppo e realizzazione di servizi e applicazioni che prevede lo sviluppo di una interfaccia applicativa (API) prima di realizzare un’applicazione o una pagina web o un’applicazione per il mobile. In altri termini, la definizione dei canali di erogazione del servizio è logicamente e cronologicamente successiva allo sviluppo dell’API. Take obbligo è previsto dal CAD all'articolo 64-bis, comma 1-bis.

Reviewer: Ciao @ggentili. Grazie per il contributo! Sicuramente la parte di contract-first che descrivi è parte della logica API-First, che però è più generale. In particolare abbiamo voluto porre qui l'accento sul fatto che i servizi vanno erogati prioritariamente tramite API, rispetto ad altre modalità (portali, file transfer e/o modalità non digitali).

Fatta salva la definizione iniziale, mi pare utile integrarla con un rimando al contract first indicato successivamente https://github.com/italia/lg-modellointeroperabilita-docs/pull/225/files#diff-ac63617ca7a652f14b6d76f12ba538e9ab275e29cdbe3255a6d8c8679bb95c15L101

@mfortini @vintra73 che ne pensate?

Author: Capisco la differenza, che può essere approfondita nel testo.... ma suggerisco di racchiudere il principio tutto sotto un unico cappello "API-first" e con un netto collegamento a quello che prevede la norma. la diffusione della cultura delle API nelle PA richiede molto tempo, quindi meglio rimanere sul concetto che, grazie al primo Piano triennale, aveva avuto un minimo di diffusione. "API-first" fa pensare alle API, "Contract-first" fa pensare al Codice appalti :-)

Reviewer: ```suggestion
     - L'API-first è un approccio in cui le PA considerano le API 
       come mezzo principale per perseguire i propri obiettivi, 
       interagendo con i propri stakeholder sin dalla fase di 
       progettazione.   
       Come indica il CAD art. 64-bis, comma 1-bis quindi,
       le interfacce applicative (API appunto) devono essere progettate
       e/o evolute in maniera interoperabile,
       a prescindere dai canali di erogazione del servizio 
       che sono individuati logicamente e cronologicamente 
       dopo la progettazione dell’API.
```

          da esso realizzati

   * -    [e-service]

>      -    I servizi realizzati da un erogatore per assicurare 

Author: cosa sono i "servizi digitali"? nel CAD non vi é definizione.

>      -    I servizi realizzati da un erogatore per assicurare 
>           l’accesso ai propri dati e/o l’integrazione dei propri processi 
>           attraverso l'interazione dei suoi sistemi informatici con 

Reviewer: ```suggestion
     -    I servizi digitali realizzati
          ai sensi del CAD art. 1, comma 1, lettera n-quater)
          da un erogatore per assicurare 
          l’accesso ai propri dati e/o l’integrazione dei propri processi 
          attraverso l'interazione dei suoi sistemi informatici con 
```

          quelli dei fruitori, trovano attuazione nell’implementazione 

>           di API. E' una categoria particolare dei servizi in rete di cui all'articolo 1, comma 1, lettera n-quater) del CAD

Reviewer: ```suggestion
          di API.
```

          da esso realizzati

   * -    [e-service]
     -    I servizi digitali realizzati
          ai sensi del CAD art. 1, comma 1, lettera n-quater)
          da un erogatore per assicurare 
          l’accesso ai propri dati e/o l’integrazione dei propri processi 
          attraverso l'interazione dei suoi sistemi informatici con 
          quelli dei fruitori, trovano attuazione nell’implementazione 

>           di API.

Reviewer: ```suggestion
          di API
```
